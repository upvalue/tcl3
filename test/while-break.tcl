while {== 1 1} {
  break
  puts bad
}

puts 42
