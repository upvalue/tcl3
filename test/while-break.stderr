{"type": "TK_ESC", "begin": 0, "end": 5, "body": "while"}
{"type": "TK_SEP", "begin": 5, "end": 6, "body": " "}
{"type": "TK_STR", "begin": 7, "end": 13, "body": "== 1 1"}
{"type": "TK_SEP", "begin": 14, "end": 15, "body": " "}
{"type": "TK_STR", "begin": 16, "end": 36, "body": "\n  break\n  puts bad\n"}
{"type": "TK_EOL", "begin": 37, "end": 39, "body": "\n\n"}
{"type": "TK_ESC", "begin": 0, "end": 2, "body": "=="}
{"type": "TK_SEP", "begin": 2, "end": 3, "body": " "}
{"type": "TK_ESC", "begin": 3, "end": 4, "body": "1"}
{"type": "TK_SEP", "begin": 4, "end": 5, "body": " "}
{"type": "TK_ESC", "begin": 5, "end": 6, "body": "1"}
{"type": "TK_EOL", "begin": 5, "end": 6, "body": "1"}
{"type": "TK_EOF", "begin": 5, "end": 6, "body": "1"}
{"type": "TK_EOL", "begin": 0, "end": 3, "body": "\n  "}
{"type": "TK_ESC", "begin": 3, "end": 8, "body": "break"}
{"type": "TK_EOL", "begin": 8, "end": 11, "body": "\n  "}
{"type": "TK_ESC", "begin": 39, "end": 43, "body": "puts"}
{"type": "TK_SEP", "begin": 43, "end": 44, "body": " "}
{"type": "TK_ESC", "begin": 44, "end": 46, "body": "42"}
{"type": "TK_EOL", "begin": 46, "end": 47, "body": "\n"}
{"type": "TK_EOF", "begin": 46, "end": 47, "body": "\n"}
